modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/ExternalRefProcessor.java

```diff
@@ -246,9 +264,29 @@ private void processSchema(Schema property, String file) {
             }
             if (property instanceof ComposedSchema) {
                 ComposedSchema composed = (ComposedSchema) property;
+                final Map<String, String> refMap = Optional.ofNullable(property.getDiscriminator())
```

Comment:
IIANM the changes here, alongside the ones in `SchemaProcessor` and `PathsProcessor`, consider `oneOf` and `anyOf` as mutually exclusive, while they can coexist even if not so common. I would suggest to consider them both in the processing

Reply:
Updated as per suggestion, thanks!